packages/firebase_analytics/firebase_analytics_web/pubspec.yaml

```diff
@@ -6,7 +6,7 @@ version: 0.5.0
 
 environment:
   sdk: '>=2.16.0 <3.0.0'
-  flutter: '>=1.12.13+hotfix.4'
+  flutter: '>=1.20.0'
```

Comment:
Is it a breaking change? Or the version number was already 1.20 in the main package?